src/test/java/com/crud/tasks/trello/client/TrelloClientTest.java

```diff
@@ -18,7 +18,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
jesli musisz uzyc silent, znaczy ze masz gdzies nieuzywane mockowanie. Mockito powinno wskazac Ci w error'ze co jest nie tak. Sprawdz uzywajac np. debug'a co jest nie tak. 

W razie czego mozemy to razem przedebuggowac na spotkaniu

Reply:
czepia się tego mocka:
when(restTemplate.getForObject(uri, TrelloBoardDto[].class)).thenReturn(null);

"Unnecessary stubbings detected in test class: TrelloClientTest
Clean & maintainable test code requires zero unnecessary code.
Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at com.crud.tasks.trello.client.TrelloClientTest.shouldReturnEmptyList(TrelloClientTest.java:92)
Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.
org.mockito.exceptions.misusing.UnnecessaryStubbingException: "